src/routeGeneration/routeGenerator.ts

```diff
@@ -122,8 +122,13 @@ export class RouteGenerator {
   }
 
   private getRelativeImportPath(fileLocation: string) {
-    fileLocation = fileLocation.replace('.ts', '');
-    return `./${path.relative(this.options.routesDir, fileLocation).replace(/\\/g, '/')}`;
+    const outputDirectory =
+      path.extname(this.options.routesDir) === '' ? // routes dir is a directory
+      this.options.routesDir :
```

Comment:
I know it's a little late to discuss options, but @HoldYourWaffle I think it would be better if `routesDir` had to be a dir and we added a new `routeFilePath` config option that had to be a `.ts` file. I find it confusing as a user to see that `routeDir` could be a file. To put it another way, I think apples should stay apples, and oranges should stay oranges.

Reply:
I agree, this was a bit of a hacked-together and rushed implementation (as mentioned by my original post).

If you feel like adding a (past-me would say "redundant") `routeFile`-specific option I'd be happy to implement that. I think there are two paths we could take for that:
1. We add a separate `routeFileName` option that changes the `routes.ts` part (my preferred option).
2. We add a `routeFilePath` option as a sort-of "sibling" (English is not my first language) to `routesDir` that serves the same purpose but without the `routes.ts` suffix. What would happen if both `routesDir` and `routeFilePath` are specified though?

Reply:
I suppose I’d be fine either option. If you selected option 2, you would just add validation that `routesDir` and `routesFilePath` were not both present. The validation would throw:

```
if(config.routesDir && config.routesFilePath) {
  throw new Error(`You can only provide routesDir or routesFilePath in your config.`);
}
```

Reply:
I implemented option 1 since it's a lot simpeler and intuitive to use (at least in my opinion)